src/main/java/org/kohsuke/github/GitHub.java

```diff
@@ -128,7 +129,13 @@
         } else {
             if (password!=null) {
                 String authorization = (login + ':' + password);
-                encodedAuthorization = "Basic "+new String(Base64.encodeBase64(authorization.getBytes()));
+                final Charset charset;
+                try {
+                    charset = Charset.forName("UTF-8");
```

Comment:
Guava is a dangerous dependency to rely on as they do not have the same backwards compat rules as Jenkins follows and you can end up breaking your plugins. Exposing core's Guava to plugins was a major mistake...